extensions/wikia/CreateNewWiki/CreateNewWikiController.class.php

```diff
@@ -97,7 +97,6 @@ public function index() {
 			'descriptionplaceholder' => wfMessage( 'cnw-desc-placeholder' )->escaped(),
 			'cnw-error-general' => wfMessage( 'cnw-error-general' )->parse(),
 			'cnw-error-general-heading' => wfMessage( 'cnw-error-general-heading' )->escaped(),
-			'cnw-keys' => $keys
```

Comment:
The $keys var is not initialized - the initialization was removed as part of SUS work ( see https://github.com/Wikia/app/commit/c5068416c5be3c4166ab0edbfb2e5ef5339a996d )